web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v447/migrate-default.sql

```diff
@@ -3,3 +3,81 @@ UPDATE Settings SET value='SNAPSHOT' WHERE name='system/platform/subVersion';
 
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system/banner/enable', 'false', 2, 1920, 'n');
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system/auditable/enable', 'false', 2, 12010, 'n');
+
+ALTER TABLE Sources ADD COLUMN datahubEnabled BOOLEAN DEFAULT FALSE;
+ALTER TABLE Sources ADD COLUMN datahubConfiguration TEXT DEFAULT '';
+
+INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('geonetwork-ui/datahub/enabled', 'false', 2, 7404, 'n');
+INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('geonetwork-ui/datahub/configuration', '
```

Comment:
Following the feedback in https://github.com/geonetwork/core-geonetwork/issues/8021#issuecomment-2301655763 you should be able to avoid using the Settings table and use the same table (Sources) for all configurations, including the default one.